.github/workflows/release.yaml

```diff
@@ -30,5 +30,7 @@ jobs:
 
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.4.0
+        with:
+          charts_dir: charts
```

Comment:
The releaser will choose the `charts` dir by default - so this shouldn't be needed.
When i use the `cr` locally - an index is created and packaging works just fine.
So I suspect this isn't needed for repo's following the helm repo structure conventions